backend/migrations/Version20250110000000.php

```
@@ -0,0 +1,68 @@
+            deployed_by VARCHAR(255) DEFAULT NULL,
+            rollback_reason LONGTEXT DEFAULT NULL,
+            rolled_back_at DATETIME DEFAULT NULL,
+            rolled_back_to_commit VARCHAR(40) DEFAULT NULL,
+            PRIMARY KEY(id),
+            INDEX idx_commit_hash (commit_hash),
+            INDEX idx_status (status),
+            INDEX idx_deployed_at (deployed_at)
+        ) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)